.github/scripts/generate_ci_workflows.py

```diff
@@ -76,56 +76,25 @@
 LABEL_CIFLOW_MACOS = "ciflow/macos"
 LABEL_CIFLOW_TRUNK = "ciflow/trunk"
 LABEL_CIFLOW_BINARIES = "ciflow/binaries"
-LABEL_CIFLOW_BINARIES_WHEEL = "ciflow/binaries/wheel"
-LABEL_CIFLOW_BINARIES_CONDA = "ciflow/binaries/conda"
-LABEL_CIFLOW_BINARIES_LIBTORCH = "ciflow/binaries/libtorch"
+LABEL_CIFLOW_BINARIES_WHEEL = "ciflow/binaries_wheel"
```

Comment:
@seemethere I had to change this, because the new push trigger matches against `ciflow/binaries/*`, which would pick up `ciflow/binaries/wheel` incorrectly. Lmk if any additional changes are necessary